packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal-title/buy-sell-modal-title.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { isDesktop } from '@deriv/shared';
+import { Localize } from 'Components/i18next';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import { buy_sell } from 'Constants/buy-sell';
+import { useStores } from 'Stores';
+
+const BuySellModalTitle = () => {
+    const { general_store, buy_sell_store, advertiser_page_store, my_profile_store } = useStores();
+    const { selected_ad_state, show_advertiser_page, table_type: buy_sell_table_type } = buy_sell_store;
+    const { showModal } = useModalManagerContext();
+
+    const { account_currency } = selected_ad_state;
+    const table_type = show_advertiser_page ? advertiser_page_store.counterparty_type : buy_sell_table_type;
+
+    if (my_profile_store.should_show_add_payment_method_form) {
+        if (isDesktop()) {
+            return (
+                <React.Fragment>
+                    <Icon
+                        icon='IcArrowLeftBold'
+                        data_testid='dt-buy-sell-modal-back-icon'
+                        onClick={() => {
+                            if (general_store.is_form_modified) {
+                                showModal({
+                                    key: 'CancelAddPaymentMethodModal',
+                                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
will discuss this with @adrienne-deriv about this 🙏

Reply:
so we needed to comment these because some of the modals that have props have not been converted to typescript yet where the modal's props can be optional, which is causing the eslint issue. But we have decided to have one final card where we would check this error and remove the comments before merging the p2p-modal-refactoring branch to master and fix it then